client/src/components/SideBar.jsx

```diff
@@ -1,37 +1,49 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import Hours from './Hours';
 
-const SideBar = (props) => {
-  const style1 = {
-    textAlign: 'center',
-  };
-  const { details } = props;
-  const { address, phoneNumber, officialWebsite } = details;
-  const {
-    street, city, zipcode, country,
-  } = address;
-  return (
-    <div style={style1}>
-      <div>
-        <span>
-          {street}
-          {city}
-          {zipcode}
-          {country}
-        </span>
-      </div>
-      <div>
-        {phoneNumber}
-      </div>
-      <div>
-        {officialWebsite}
-      </div>
+class SideBar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+    };
+  }
+
+  render() {
+    const { details } = this.props;
+    const {
+      openHours, closeHours, address, phoneNumber, officialWebsite,
```

Comment:
Nothing wrong with the destructuring here, but we can do a little bit more.

Reply:
I refactored it!